tests/integration/mypy_test.py

```diff
@@ -1 +1 @@
-import vcr_cleaner
+import vcr_cleaner  # noqa: F401
```

Comment:
Is this file useful? If yes, please add a comment in the file.

Reply:
I have no idea. I added the pragma in order to pass lint, figuring we could discuss it here.

Reply:
Here's where this file was introduced: 
https://github.com/techservicesillinois/vcrpy-cleaner/pull/10/files

I think what is happening is that doing the import ensures that MyPy typing is enforced.

Reply:
I'm curious whether we could get the same impact by changing:

```make
integration: .integration.venv
	$(TENV_PYTHON) -m mypy tests/integration/mypy_test.py
```

to

```make
integration: .integration.venv
	$(TENV_PYTHON) -m mypy src/vcr_cleaner/__init__.py
```

Reply:
I made this update, and then made appropriate updates required by MyPy - since the change apparently allows it to see more of the code.